gtfs/spec/en/reference.md

```diff
@@ -166,7 +166,7 @@ File: **Required**
+|  `route_type` | Enum | **Required** | Indicates the type of transportation used on a route. Valid options are: <br><br>`0` - Tram, Streetcar, Light rail. Any light rail or street level system within a metropolitan area.<br>`1` - Subway, Metro. Any underground rail system within a metropolitan area.<br>`2` - Rail. Used for intercity or long-distance travel.<br>`3` - Bus. Used for short- and long-distance bus routes.<br>`4` - Ferry. Used for short- and long-distance boat service.<br>`5` - Cable tram. Used for street-level rail cars where the cable runs beneath the vehicle (e.g., cable car in San Francisco).<br>`6` - Aerial lift, suspended cable car (e.g., gondola lift, aerial tramway). Cable transport where cabins, cars, gondolas or open chairs are suspended by means of one or more cables.<br>`7` - Funicular. Any rail system designed for steep inclines.<br>`11` - Trolleybus. Electric buses that draw power from overhead wires using poles.<br>`12` - Monorail. Railway in which the track consists of a single rail or a beam. <br><br>In addition, an extension has been proposed to allow specifying several route types using a relevant subset of Hierarchical Vehicle Type (HVT) codes from the European TPEG standard. <br><br>`100` - Railway Service<br>`101` - High Speed Rail Service<br>`102` - Long Distance Trains<br>`103` - Inter Regional ail Service<br>`105` - Sleeper Rail Service<br>`106` - Regional Rail ervice<br>`107` - Tourist Railway Service<br>`108` - Rail Shuttle (Within Complex)<br>`109` - Suburban Railway<br>`200` - Coach Service<br>`201`  - International Coach Service<br>`202` - National Coach Service<br>`204` - Regional Coach Service<br>`400` - Urban Railway Service<br>`401` - Metro Service<br>`402` - Underground Service<br>`403` - Urban Railway Service<br>`405` - Monorail<br>`700` - Bus Service<br>`701` - Regional Bus Service<br>`702` - Express Bus Service<br>`704` - Local Bus Service<br>`715` - Demand and Response Bus Service<br>`800` - Trolleybus Service<br>`900` - Tram Service<br>`901`  - City Tram Service<br>`902` - Local Tram Service<br>`903` - Regional Tram Service<br>`904` - Sightseeing Tram ervice<br>`905` - Shuttle Tram Service<br>`906` - All Tram Services<br>`1000` - Water Transport Service<br>`1200` - Ferry Service<br>`1300` - Aerial Lift Service<br>`1400` - Funicular Service<br>`1501` - Communal Taxi Service<br>`1700` - Miscellaneous Service<br>`1702` - Horse-drawn Carriage|
```

Comment:
I don't support these kind of changes. Google Maps is not the exclusive user of these datasets.

Reply:
From cursorary observation of a small sample of datasets, these seem to be the more popular ones. 

Very happy to revert that commit if these (official but unofficial) extensions are actually in use in the real world.

Reply:
I would say, these values should be update to recent TPEG changes, and then use all the TPEG values. It is a hierarchy anyway.

Reply:
If it's desirable to synchronise with TPEG2-PTS (TS 21219-13:2020), that seems like a logical course of action.  However, I don't personally have access to this specification (it comes at a [cost of 178 CHF from the ISO webshop](https://www.iso.org/obp/ui#iso:std:iso:ts:18234:-5:ed-1:v1:en) or [£260 from the BSI webshop](https://shop.bsigroup.com/ProductDetail?pid=000000000030114290)) to know the new values, and as far as I know they aren't publically available in isolation from the standard anywhere. I'm also unsure on the terms of the copyright for that standard and whether it'd be permissible to replicate a part of it in the GTFS specification.  

However, it may not be desirable to simply replicate the TPEG standard list of vehicle types, but actually only a subsection of it that is likely to be useful for data export in GTFS format. The intended uses of the two are different - TPEG is primarily interested in transmission of transport related information for immediate consumer use (e.g. traffic alerts in DAB radios, where it originated) - rather than describing a schedule or changes to it. The types of vehicle represented may not all be relevant to GTFS. 

Then again, transit providers may already have systems which use the TPEG codes for use in different areas (e.g. issuing service alerts to broadcasters), so making the two instantly compatible may be beneficial. 

Hard for me to know from my viewpoint! Would welcome thoughts from others.

Reply:
I think SIRI should be / is going to be updated. Already with a limited scope.

Reply:
[SIRI modes](https://github.com/SIRI-CEN/SIRI/blob/6f218682c5d69e57cafdc6ca78e5e3a2f915e1f7/xsd/siri_model/siri_modes.xsd) for the most part seem to be compatible here. The only difference is they don't have numeric values (they've just enums) - so in order to preserve compatibility with providers implementing the defacto Google extension, the original numbers would need to be preserved and any additions to be assigned new numbers.